sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -71,23 +84,30 @@ export interface AspectSentiment {
 export { AzureKeyCredential }
 
 // @public
-export type BeginAnalyzeHealthcareOperationState = PollOperationState<PaginatedHealthcareEntities>;
+export interface BeginAnalyzeBatchTasksOptions {
+    analyze?: AnalyzeJobOptions;
+    polling?: PollingOptions;
+}
 
 // @public
 export interface BeginAnalyzeHealthcareOptions {
-    health?: HealthcareJobOptions;
+    healthcare?: HealthcareJobOptions;
     polling?: PollingOptions;
 }
 
 // @public
-export type BeginAnalyzeOperationState = PollOperationState<PaginatedAnalyzeResults>;
+export interface BeginAnalyzeHealthcarePollState extends AnalysisPollOperationState<PaginatedHealthcareEntities> {
```

Comment:
not much knowledge on the JS guidelines for LROs but at least from what I see in the [samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/core-lro/samples#implementing-an-operation) and in the [FR LROs](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md) the `xxState` objects don't contain the prefix `Begin` and have suffix `OperationState` instead of `PollState`.

Reply:
Same comment will apply for Analyze method

Reply:
@sadasant @witemple-msft, do we have a standard naming for these?

Reply:
I think the latest we've got to a baseline for naming is Key Vault Certificates: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

One example could be:

```ts
export interface KeyVaultCertificatePollOperationState<TResult> extends PollOperationState<TResult> {
    certificateName: string;
}
```

Remember that these operation states are intended to only represent the public surface of the operation state, so the closer they are to the base `PollOperationState`, the better.